File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.py
def solution(s):
    words = ['zero','one','two','three','four','five','six','seven','eight','nine']
    numbers = [0,1,2,3,4,5,6,7,8,9]
    answer = s
    for i in range(len(words)):
        if answer.find(words[i]) != -1:
            answer = answer.replace(words[i],str(numbers[i]))
    return int(answer)
    

File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.js
function solution(s) {
    var arr = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']

    arr.forEach((e, i) => {
        s = s.split(e).join(i)        
    })

    return Number(s);
}